src/image_rgb.c

```diff
@@ -160,8 +119,11 @@ static frame *rgb_copy(frame *src, frame *dst)
   dst->width = src->width;
   dst->height = src->height;
   dst->stride = src->stride;
-  dst->data = memalign(ALIGNMENT_BYTES, Rgb_data_size(src));
```

Comment:
No more aligned memory?

Reply:
Was it really useful? The constant `ALIGNMENT_BYTES` seemed pretty arbitrary to me...